stream_test.go

```diff
@@ -0,0 +1,20 @@
+package eventsource
+
+import (
+	"fmt"
+	"testing"
+	"time"
+)
+
+// This particular "benchmark" exists to spit out various jitter values. It's structured
```

Comment:
`test -v` will also show any stderr/std out so you don't need `T.Logf`. It seems weird to have a loop in a benchmark that doesn't loop `b.N` times, but whatever.

Maybe a better test would be to get a bunch of jittered backoff values, average them, and assert that it is within some threshold of a mean?